README.md

```diff
@@ -1,23 +1,44 @@
 # Best Practices for Discord Bot Lists
-*Bot lists must meet these minimum requirements to be considered a bot list.*
 
 ---
 
-Your bot list:
+*Bot lists must meet these minimum requirements to be considered a bot list.*
+
+## Your bot list must:
 
-1. Must use a custom domain name with a respectable, paid TLD.
-2. Must not be a `[xyz].glitch.me` domain.
+1. use a custom domain name with a respectable, paid TLD.
+2. not be a `[xyz].glitch.me` domain.
    - Glitch is acceptable if:
       - Uptime is immaculate
       - A custom domain name is used (whilst still following point `1`)
-3. Should not display all bots on the homepage.
-4. Must be fully functioning.
+3. be fully functioning.
    - No broken links/buttons.
-5. Must have consistent uptime.
+4. have consistent uptime.
    - Minor downtime for fixing a problem is fine
 
 ---
 
+*Bot lists should meet these requirements to be a good bot list.*
+
+## Your bot list should:
+
+1. not display all bots on the homepage.
+2. not be hosted on Glitch.
+3. have an API with good documentation.
+   - API should support `GET`ting bot information
+      - Uses a JSON body format for data
+   - API should support `POST`ing bot stats
+      - Server/guild count
+      - Shard support
+      - Uses an Authorization header
+      - Uses a JSON body format for data
+4. respond with correct status codes.
+   - All pages that load correctly should respond with 200
```

Comment:
Maybe change to 2xx? Since there are other 200 codes that would also make sense?